config/routes.rb

```diff
@@ -18,6 +18,7 @@
       default: true) do
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Metrics/LineLength
**Severity**: INFO
**File**: config/routes.rb L18

Line is too long. [84/80]

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*